### ISSUE TITLE
Accessibility: Enter on Extension Item doesn't focus the Extension Editor

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#91603]
Re #91063

- OS: Ubuntu 18.04

Bug:

* Turn on Screen Reader
* Navigate to Extensions Viewlet
* Navigate to an extension item, press enter, focus doesn't go into the extension editor.

**Comment by isidorn** [comment]
Yeah, we should change this.
Forwarding to @sandy081 
The fix should be pretty simple, on list open we should pass focus to the extensionds details view

**Comment by sandy081** [comment]
Its not trivial as the list does not have enough details. After talking to @joaomoreno need to do some changes in list. So moving to March.

**Comment by isidorn** [comment]
Thanks for fixing this. Adding bug label so we get this verified.
